src/test/poms/hadoop/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is possible to automatically run these tests using the invoker plugin. This is how maven plugin ITs are typically executed. I think there may be a few examples in some of the other revelc maven plugins.

Reply:
Should open a follow on issue for that.